Prose/ProseLib/Sources/ConversationFeature/Chat/Chat.swift

```diff
@@ -199,11 +199,14 @@ struct ChatView: NSViewRepresentable {
       // but still allow starting with a non-nil value (which `dropFirst()` would prevent).
       .drop(while: { $0 == nil })
       .removeDuplicates()
-      .sink { menuState in
+      .sink { [weak coordinator = context.coordinator] menuState in
+        guard let coordinator = coordinator else {
+          return
+        }
         if let menuState: MessageMenuState = menuState {
-          self.showMenu(menuState, on: webView, context: context)
+          self.showMenu(menuState, on: webView, coordinator: coordinator)
```

Comment:
Isn't it a problem that `webView` is sent here? Shouldn't we add `weak webView` too?

Reply:
No ;)

Reply:
The answer is no: The coordinator was captured, which prevented the `AnyCancellable`s from being canceled. Now they are, which frees the web view.